packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -187,30 +188,40 @@ export class TablePageObject<T extends TableRecord> {
         return anchor as Anchor;
     }
 
-    public getRenderedIconColumnCellIcon(
+    public getRenderedIconColumnCellContentElement(
```

Comment:
Calling the icon/spinner the "Content Element" makes these function names a bit long & awkward to read. What do you think about just calling it an "Icon" even though it could be an icon or a spinner? Technically we called it the "icon" column, so maybe it isn't too confusing to extend that terminology here even though it might be a spinner rather than an icon. That would make these functions `getRenderedIconColumnCellIcon` and `getRenderedIconColumnGroupHeaderIcon`. Thoughts?